examples/2d/change_sprite_runtime.rs

```diff
@@ -0,0 +1,63 @@
+use bevy::{
+    core::{Time, Timer},
+    ecs::prelude::*,
+    math::Vec3,
+    prelude::{App, AssetServer, Handle, Transform},
+    render2::{camera::OrthographicCameraBundle, color::Color, texture::Image},
+    sprite2::{PipelinedSpriteBundle, Sprite},
+    PipelinedDefaultPlugins,
+};
+
+fn main() {
+    App::new()
+        .add_plugins(PipelinedDefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(change_texture)
+        .add_system(change_color)
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    let texture_handle = asset_server.load("branding/bevy_logo_dark.png");
+    let texture_handle_2: Handle<Image> = asset_server.load("branding/bevy_logo_light.png");
+    commands.spawn_bundle(OrthographicCameraBundle::new_2d());
+    commands
+        .spawn_bundle(PipelinedSpriteBundle {
+            texture: texture_handle.clone(),
+            transform: Transform {
+                translation: Vec3::new(1., 1., 1.),
+                scale: Vec3::ONE,
+                ..Default::default()
+            },
+            sprite: Sprite {
+                color: Color::WHITE,
+                ..Default::default()
+            },
+            ..Default::default()
+        })
+        .insert(Timer::from_seconds(1.5, false));
+
+    commands.insert_resource(texture_handle_2);
+}
+
+fn change_texture(
+    time: Res<Time>,
+    texture: Res<Handle<Image>>,
```

Comment:
Storing raw-ish types like this is prone to conflict in surprising ways. In real code, this should be something like a `BevyLogoLight` handle, which stores this type.

We should swap to that pattern to promote idiomatic code and more clearly show what's going on.